packages/server/src/complete/candidates/createTableCandidates.ts

```diff
@@ -10,20 +10,54 @@ import { ICONS } from '../CompletionItemUtils'
  * @returns
  */
 function allTableNameCombinations(table: Table): string[] {
```

Comment:
improvement: Just repeating potential improvement from your intro video, this could be returning just a string

EDIT: Would also finding a more appropriate name for the function be an overkill?

Reply:
Absolutely, refactoring!